integreat_cms/cms/templates/statistics/statistics_sidebar.html

```diff
@@ -0,0 +1,52 @@
+{% load i18n %}
+{% load static %}
+{% load widget_tweaks %}
+<div class="rounded border border-solid border-blue-500 shadow-2xl bg-white hidden mb-2"
+     id="chart-legend-container">
+    <div class="rounded p-4 bg-water-500">
+        <h3 class="heading font-bold text-black">
+            <i icon-name="settings" class="pb-1"></i> {% translate "Adjust shown data" %}
+        </h3>
+    </div>
+    <div id="chart-legend">
+        {% if request.region.statistics_enabled and perms.cms.view_statistics %}
```

Comment:
```suggestion
```
The same check is already performed in `statistics_overview.html`, right before importing this file.

Actually, maybe wrapping the entire contents of `statistics_overview.html` (i.e., everything in the `content` block) would be an option and allow you to remove all other checks?

Reply:
yes, you are right :+1: Thank you for bringing this up :)